jnigen/lib/src/generate_bindings.dart

```diff
@@ -31,12 +31,12 @@ Future<void> generateJniBindings(Config config) async {
   final mavenDl = config.mavenDownloads;
   if (mavenDl != null) {
     final sourcePath = mavenDl.sourceDir;
-    Directory(sourcePath).create(recursive: true);
+    await Directory(sourcePath).create(recursive: true);
```

Comment:
Nice catch, this should have created crashes earlier.

Reply:
unawaited_futures should be default lint rule or something.